src/components/pageNav/index.jsx

```diff
@@ -26,14 +26,15 @@ class PageNav extends React.PureComponent {
         icon
         className="outlined"
         to={buttonLink}
-        component={Link}
+        component={type === 'previous' ? Link : disableButton ? null : Link}
         iconEl={
           type === 'previous' ? (
             <ButtonIcon srText={title || 'Home'} Icon={ArrowLeft} />
           ) : (
             <ButtonIcon srText={title || 'Home'} Icon={ArrowRight} />
           )
         }
+        disabled={type === 'previous' ? false : disableButton}
```

Comment:
so with the above change is always false for previous so we can simplify to `disabled={disableButton}`